app/src/main/java/com/github/se/polyfit/ui/components/Graphs.kt

```diff
@@ -0,0 +1,81 @@
+package com.github.se.polyfit.ui.components
+
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.yml.charts.axis.AxisData
+import co.yml.charts.common.extensions.formatToSinglePrecision
+import co.yml.charts.ui.linechart.model.GridLines
+import co.yml.charts.ui.linechart.model.IntersectionPoint
+import co.yml.charts.ui.linechart.model.Line
+import co.yml.charts.ui.linechart.model.LineChartData
+import co.yml.charts.ui.linechart.model.LinePlotData
+import co.yml.charts.ui.linechart.model.LineStyle
+import co.yml.charts.ui.linechart.model.LineType
+import co.yml.charts.ui.linechart.model.SelectionHighlightPoint
+import co.yml.charts.ui.linechart.model.SelectionHighlightPopUp
+import co.yml.charts.ui.linechart.model.ShadowUnderLine
+import com.github.se.polyfit.ui.viewModel.DataToPoints
+
+@Composable
+@Preview
+fun lineChartData(): LineChartData {
+  val steps = 10
+  val points = DataToPoints()
```

Comment:
doing it in 2 separate funcitons won't add anything as this is the data for the composable not the actual composable, so the preview won't show anything anyway so I'll take it out. I will however put the datapoints as parameters for the function

Reply:
Okay, I see, but it might be a good idea to follow this pattern when you have @Preview to make the code more readable and facilitate integration with the class that will retrieve data for the graphs.